Sources/AblyChat/DefaultMessages.swift

```diff
@@ -0,0 +1,265 @@
+import Ably
+
+
+public struct MessageEventPayloadWrapper: Hashable, Sendable {
+    let id: String
+    let listener: @Sendable (MessageEventPayload) -> Void
+
+    // Hashable conformance via the id
+    public func hash(into hasher: inout Hasher) {
+        hasher.combine(id)
+    }
+
+    public static func == (lhs: MessageEventPayloadWrapper, rhs: MessageEventPayloadWrapper) -> Bool {
+        return lhs.id == rhs.id
+    }
+    
+    public init(id: String, listener: @Sendable @escaping (MessageEventPayload) -> Void) {
+        self.id = id
+        self.listener = listener
+    }
+}
+
+typealias fromSerial = String
```

Comment:
What's the purpose of this typealias? Also its name should be capitalised.

Reply:
added a comment